v5/securityandcompliancecenterapiv3/security_and_compliance_center_api_v3_integration_test.go

```diff
@@ -1490,7 +1490,7 @@ var _ = Describe(`SecurityAndComplianceCenterApiV3 Integration Tests`, func() {
 			Expect(response.StatusCode).To(Equal(200))
 			Expect(providerTypesCollection).ToNot(BeNil())
 
-			providerTypeIdLink = *providerTypesCollection.ProviderTypes[0].ID
+			providerTypeIdLink = *providerTypesCollection.ProviderTypes[1].ID
```

Comment:
```suggestion
			for _, providerType := range *providerTypesCollection.ProviderTypes {
			    if providerType.Name == "workload-protection" {
			        providerTypeIdLink = *providerType.ID
			    }
			}
```

if it's possible, we should change this block of code to look for the name 'workload-protection'. I don't know if the swagger tags has this type of functionality.